backend/worker/kafka_worker.go

```diff
@@ -127,7 +127,13 @@ type KafkaWorker struct {
 }
 
 func (k *KafkaBatchWorker) flush(ctx context.Context) error {
-	s, _ := util.StartSpanFromContext(ctx, util.KafkaBatchWorkerOp, util.ResourceName(fmt.Sprintf("worker.kafka.%s.flush", k.Name)))
+	log.WithContext(ctx).WithFields(
+		log.Fields{
+			"worker_name":    k.Name,
+			"message_length": len(k.messages)},
+	).Info("KafkaBatchWorker flushing messages")
```

Comment:
how often does this happen? might want to sample (ie. how we do it in the `log` method)